examples/capture_image_full_res.py

```diff
@@ -17,7 +17,6 @@
 image = picam2.switch_mode_and_capture_image(capture_config)
 image.show()
 
-
```

Comment:
Could this whole commit be a separate pull request too, please? Sorry that this is a bit of a hassle, but it's not really related to the other changes here, could be merged earlier, and would leave less in this PR to review. Thanks!

Reply:
This was part of style cleanup associated with PEP8 that was run on everything, not just my additions. I can revert that commit and focus on only my stuff if you would like.

Reply:
Yes please, when there's a long list of commits to review it's helpful if other stuff doesn't creep in, though I know how tempting that always is!